modules/custom-status/lib/configure.scss

```diff
@@ -2,21 +2,29 @@
 	display: flex;
 }
 
-// Draw background line connecting statuses, add overlay background to areas where the line should be hidden
+.status-start,
```

Comment:
Why did these rules change order in this file?

Reply:
It's because it was trying to enforce that, if you have a CSS rule targeting one element then do that first followed by targeting any element inside it. So if you notice, `.add-status` is targeted by itself and then within `.status-container` as well. So it's saying, do the outer one first and then the inner one. It's the same thing for the other rules as well.

From what I read, it's supposed to also make you re-think those CSS rules. So for us, we do have two different containers surrounding the draggable statuses that would be good to simplify. But that's been a bit buggy so we have left it for now, would be good to revisit that at some point so it'd be just one container that we have. Then this CSS would be simpler.

Hope all that made sense